mne/viz/topo.py

```diff
@@ -152,6 +153,11 @@ def _plot_topo(info, times, show_func, click_func=None, layout=None,
     """Helper function to plot on sensor layout."""
     import matplotlib.pyplot as plt
 
+    if layout.kind == 'custom':
+        layout = copy(layout)
+        for i_dim in layout.pos.T[:2]:
+            i_dim -= i_dim.min(0)
+            i_dim /= (i_dim.max(0) - i_dim.min(0))
```

Comment:
Then this should be fine, yeah. But note that after subtracting the `min` you can just divide by the `max` (instead of `max - min`). You can also do this with vectorized arithmetic/axes instead of iterating over dimensions.